Tests/UnitTests/Purchasing/ProductsFetcherSK1Tests.swift

```diff
@@ -148,8 +148,9 @@ class ProductsFetcherSK1Tests: TestCase {
             completionCallCount += 1
         }
 
+        let timeout = (productsRequestResponseTime + .milliseconds(30)).nimbleInterval
```

Comment:
I see that below you're doing `.seconds.nimbleInterval` when coming from `milliseconds`, just double-checking this is correct

Reply:
Actually this is a build error. Trying to fix lint I forgot to add the `seconds`. Good catch!